Dockerfile

```diff
@@ -39,7 +39,9 @@ ENV \
     JCLOUDS_KEYSTONE_SCOPE="" \
     JCLOUDS_KEYSTONE_PROJECT_DOMAIN_NAME=""
 
-EXPOSE 80
+EXPOSE 8080
```

Comment:
Won't this break the configuration for any users currently relying on port 80?

Reply:
That's just the internal service, Docker users currently depending on the service being available on port 80 just have to map 80:8080. So if this PR is merged, a little update to the README would be a good idea.

Reply:
Lots of people deploy S3Proxy in a variety of CI environments.  I don't want to break anyone's existing configuration.  Is there any way to transition cleanly?

Reply:
It's possible to declare 'EXPOSE' for 80 and 8080 in the Dockerfile to document the options and then via an ENV variable tell the java process which port to listen on. That way 80 could remain the default to prevent a breaking change.